pgjdbc/src/main/java/org/postgresql/util/ByteConverter.java

```diff
@@ -157,14 +160,18 @@ public static Number numeric(byte [] bytes, int pos, int numBytes) {
       throw new IllegalArgumentException("invalid length of bytes \"numeric\" value");
     }
 
-    if (!(sign == NUMERIC_POS
-        || sign == NUMERIC_NEG
-        || sign == NUMERIC_NAN)) {
-      throw new IllegalArgumentException("invalid sign in \"numeric\" value");
-    }
-
-    if (sign == NUMERIC_NAN) {
-      return Double.NaN;
+    switch (sign) {
```

Comment:
should modify the comment at line 132 indicating possible things the sign value can represent

Reply:
Done